torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -386,6 +396,314 @@ at::Tensor tensor_from_cuda_array_interface(PyObject* obj) {
       at::device(kCUDA).dtype(dtype)
   );
 }
+
+
+template <typename T>
+std::string get_vector_str(std::vector<T> & vec, int64_t start, int64_t end=-1) {
```

Comment:
Since end is never used let's simplify by leaving it out

Reply:
I think its good that ```end``` is there so that the function is general enough and might provide useful for any future use.